thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/MergedAnomalyResultBean.java

```diff
@@ -2,29 +2,33 @@
 
 import com.linkedin.thirdeye.api.DimensionMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 import org.apache.commons.lang.ObjectUtils;
 
 
-public class MergedAnomalyResultBean extends AbstractBean
-    implements Comparable<MergedAnomalyResultBean> {
+public class MergedAnomalyResultBean extends AbstractBean implements Comparable<MergedAnomalyResultBean> {
   private Long functionId;
   private Long anomalyFeedbackId;
-  private List<Long> rawAnomalyIdList;
   private String collection;
   private String metric;
   private DimensionMap dimensions;
-  private Long startTime;
-  private Long endTime;
-  // significance level
-  private double score;
-  // severity
-  private double weight;
+  private long startTime;
+  private long endTime;
+
+  private double avgCurrentVal; // actual value
+  private double avgBaselineVal; // expected value
+  private double score; // confidence level
+  private double weight; // change percentage, whose absolute value is severity
+
+  private Map<String, String> context; // additional anomaly detection result
```

Comment:
is this like the counterpart for properties in previous RawAnomalyResult? should we take the same naming convention (this might make the migration easier)?